wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
```

Comment:
There are a handful of cases that I think of where there isn't a 1 to 1 replacement.

For example the term master. In some cases the replacement should be "main" and others "controller". For slave, there is likely a more specific term like "node", "agent", or "daemon" depending on the entity being renamed.

So perhaps this should change to:

```md
# Recommendation: replace Old Term

Replacements: 
- `new term`: description of where this term is applicable.
<any additional terms>
```

Reply:
I personally prefer this approach for master, it's a very overloaded term, and an appropriate replacement is context sensitive (eg: https://developers.google.com/style/word-list#master)

Reply:
I like the google's guide @spiffxp tagged. And yes there wouldn't be 1:1 replacement for all the terminologies.

Reply:
Awesome!! Thank you so much @spiffxp! 

I'm going to add the `Replacements` block under the 2nd-level heading for recommendation, but i think it's solid!